torch/_dynamo/output_graph.py

```diff
@@ -1556,6 +1563,11 @@ def example_inputs(self) -> list[torch.Tensor]:
         result = [arg.example for arg in self.graphargs]
         return result
 
+    def remove_unused_get_attr_nodes(self) -> None:
+        for node in reversed(list(self.graph.nodes)):
+            if node.op == "get_attr" and len(list(node.users)) == 0:
```

Comment:
```suggestion
        for node in sorted(self.graph.find_nodes(op="get_attr"), reverse=True):
            if len(list(node.users)) == 0:
```
So we don't need to take a pass over the entire graph.

Reply:
Not fixed?